pyuvdata/uvbeam.py

```diff
@@ -1106,7 +1107,9 @@ def interp(self, az_array=None, za_array=None, freq_array=None, freq_interp_tol=
         return getattr(self, interp_func)(az_array, za_array, freq_array,
                                           freq_interp_kind=self.freq_interp_kind,
                                           freq_interp_tol=freq_interp_tol,
-                                          polarizations=polarizations, **kwargs)
+                                          polarizations=polarizations,
+                                          freq_interp_kind=freq_interp_kind,
```

Comment:
why is `freq_interp_kind` coming in here twice?

Reply:
that was a typo that I fixed and is now outdated, I believe..